dao/src/main/java/se/fortnox/reactivewizard/db/ReactiveStatementFactory.java

```diff
@@ -85,6 +93,19 @@ public Object create(Object[] args, ConnectionScheduler connectionScheduler) {
             });
         }
 
+        // On the possibility that FluxSink signals next and then an
```

Comment:
If there was an insert, and the database returned `1` rows affected, but the connection could not be closed after that due to network error or similar, then the new behaviour would be to return an error? Wouldn't that be wrong? The caller would assume that an error means that the insert was not successful?

I may have misunderstood this. Cannot find any new test that asserts there was an error either.

Reply:
I suggest to never return an error for connection problems that happens after all data has been returned. Connections are pooled, so you could argue that the closing of the connection has nothing to do with the query.

Was there any new scenario that caused you to change the behaviour here?